tslint.json

```diff
@@ -36,25 +34,16 @@
         "no-empty-interface": false,
         "no-unused-expression": false,
         "object-literal-sort-keys": false,
-        "one-line": [
-            false
-        ],
-        "one-variable-per-declaration": [
-            false
-        ],
-        "ordered-imports": [
-            false
-        ],
+        "one-line": false,
+        "one-variable-per-declaration": false,
+        "ordered-imports": false,
         "quotemark": [
             true,
             "single"
         ],
-        "semicolon": [
-            true
-        ],
-        "variable-name": [
-            false
-        ]
+        "semicolon": true,
+        "variable-name": false,
+        "whitespace": [true, "check-module"]
```

Comment:
This is the only actual addition in this file. The rest of the changes are from making spacing consistent.